### PR TITLE
Migrate first batch of forms

### PR DESCRIPTION
**Pull request by jbalboni**
This migrates the VIC v2, VRE, and Veteran Representative forms to the external forms system. None of these forms are live, so it's not very risky.

### CODE REVIEW

**Review by annekainicUSDS — APPROVED** [review]
LGTM! Do we want to wait to merge this until the HCA changes are deployed and we can ensure that's working well? Or are you comfortable merging this now?

**Comment by annekainicUSDS** [comment]
One other thought I had: do we also want to update the scroll functions for the e2e tests for these? I noticed your tests are all passing without that change, but the edu forms I migrated last night weren't. I wonder if this might be a problem in the future that we may as well fix now?

**Comment by jbalboni** [comment]
Ah, I should definitely update those tests. These forms are all disabled in prod, so the e2e tests aren't running in the build.

I think we can wait until HCA looks good to merge this.

**Pull request merged**
